14/umbraco-cms/tutorials/creating-your-first-extension.md

```diff
@@ -254,17 +216,18 @@ If you create multiple dashboards it's necessary to change the alias of `@custom
 
 The code above defines a Web Component that contains a button that when clicked will open a notification with a message to the user.
 
-6. Build the `ts` file at the root of the `my-typescript-extension` folder so that we can use it in our package:
+4. Build the `ts` file at the root of the `Client` folder so that we can use it in our package:
 
 ```bash
 npm run build
 ```
 
-After running the build, you will see a new file in the `dist` folder with the name `my-typescript-extension.js`. This is the file we will use in our package.
+After running the build, you will see a new file in the `App_Plugins/Client` folder with the name `client.js`. This is the file we will use in our package.
 
-7. Create an `umbraco-package.json` file at the root of the `my-typescript-extension` folder with the following code:
+5. If you followed the Vite article, you will already have an `umbraco-package.json` file in the `Client/public` folder. If not, please create it first. Now paste in the following:
```

Comment:
Line 25 of **this article**, tells you to create the `umbraco-package.json` file 🤔

Reply:
You are right, we could probably remove this step.